tests/ga/test_extension.py

```diff
@@ -3527,6 +3527,23 @@ def test_handler_manifest_boolen_fields(self):
         self.assertFalse(manifest.is_report_heartbeat())
         self.assertFalse(manifest.supports_multiple_extensions())
 
+    def test_unicode_values_for_boolean_fields_in_handler_manifest(self):
+        handler_json = {
```

Comment:
We should do this test loading the json from a file, which is what exposed the problem

Other similar tests should probably also be modified